extensions/wikia/FeedsAndPosts/WikiInfo.class.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+use CommunityDataService;
+use Title;
+
+class WikiInfo {
+	public function get(): array {
+		global $wgCityId, $wgEnableCommunityPageExt;
+
+		$getStartedUrl = !empty($wgEnableCommunityPageExt)
```

Comment:
Minor: our PHP guidelines call for padding in all of these function calls with arguments.